xcube/core/verify.py

```diff
@@ -81,9 +81,38 @@ def verify_cube(dataset: xr.Dataset) -> List[str]:
     if xy_var_names and time_var_name:
         _check_data_variables(dataset, xy_var_names, time_var_name, report)
 
+    if xy_var_names:
+        _check_coord_equidistance(dataset, xy_var_names[0], xy_var_names[0], report)
+        if xy_var_names[0] + '_bnds' in dataset.variables:
+            _check_coord_equidistance(dataset, xy_var_names[0] + '_bnds', xy_var_names[0], report)
+        _check_coord_equidistance(dataset, xy_var_names[1], xy_var_names[1], report)
+        if xy_var_names[0] + '_bnds' in dataset.variables:
+            _check_coord_equidistance(dataset, xy_var_names[1] + '_bnds', xy_var_names[1], report)
+
     return report
 
 
+def _check_coord_equidistance(dataset, coord_name, dim_name, report, rtol=1e-09):
+    coordinate = dataset[coord_name]
+
+    if 'bnds' in coord_name:
```

Comment:
Use function that determines the bounds variable of a coordinate variable.   
`coord_var_bnds_name = coord_var.attrs.get("bounds", coord_var_name + "_bnds")`